README.md

```diff
@@ -1,18 +1,26 @@
 # RTI BioCloud Docker Library
 
-Welcome to the RTI BioCloud Docker Library! This repository serves as a central location for Docker images used by RTI's cloud-based bioinformatics toolkit.
+Welcome to the RTI BioCloud Docker Library! This repository serves as a central location for building and cataloging the Docker images used by RTI's cloud-based bioinformatics toolkit.
 
 ## Repository Structure
 
 The Dockerfiles are organized in the following structure: `<tool-name>/<tool-version>/Dockerfile`. Each tool has its own directory containing the Dockerfile and related files.
```

Comment:
We've been following the convention of using underscores to separate words in a tool. Let's replace this hyphen with an underscore.

Reply:
I have updated the readme to reflect this convention